helm/ngrok-operator/values.yaml

```diff
@@ -6,11 +6,13 @@
 ## @param description ngrok-operator description that will appear in the ngrok dashboard
 ## @param commonLabels Labels to add to all deployed objects
 ## @param commonAnnotations Annotations to add to all deployed objects
+## @param oneClickDemoMode If true, then the operator will startup without required fields or API registration, become Ready, but not actually be running
 nameOverride: ""
 fullnameOverride: ""
 description: "The official ngrok Kubernetes Operator."
 commonLabels: {}
 commonAnnotations: {}
+oneClickDemoMode: false
```

Comment:
doesn't this need to be set to true in order for it to work without having to set any required fields? Or do these on click app marketplaces allow specifying a helm value like this by default and its just we can't require user specific values to be set like API crednetials?

Reply:
>  Or do these on click app marketplaces allow specifying a helm value like this by default and its just we can't require user specific values to be set like API crednetials?

The latter: My understanding is we can provide a `values.yaml` but not a pop-up for required fields. So we'll set `onClickDemoMode: true` in those marketplace values.

Reply:
ahh ok, thats good to know, thanks

Reply:
if we can provide default values i'm kinda surprised we didn't just default these installations' replica count to 0 and instruct you to scale up after provider credentials vs a no-op mode, but I don't think I have all the historic context here